Source/WarpX.H

```diff
@@ -1281,7 +1281,7 @@ private:
 
     int regrid_int = -1;
 
-    amrex::Real cfl = amrex::Real(0.7);
+    amrex::Real cfl = amrex::Real(0.99);
```

Comment:
Just in case: I usually use `0.999` for all my sims, which is a bit closer not leaving 1% on the table for time steps and still far from machine precision to be rounded to 1.
```suggestion
    amrex::Real cfl = amrex::Real(0.999);
```

I base this on Figure 2.11 in https://doi.org/10.5281/zenodo.3266820 & Figure 2.10 in https://doi.org/10.5281/zenodo.15924, where I still find the difference is signficant for the third digit.